ql/src/java/org/apache/hadoop/hive/ql/io/parquet/vector/VectorizedParquetRecordReader.java

```diff
@@ -248,9 +245,12 @@ public void initialize(
     requestedSchema = DataWritableReadSupport
       .getRequestedSchema(indexAccess, columnNamesList, columnTypesList, fileSchema, configuration);
 
-    Path path = wrapPathForCache(filePath, cacheKey, configuration, blocks, cacheTag);
+    //TODO: For data cache this needs to be fixed and passed to reader.
+    //Path path = wrapPathForCache(filePath, cacheKey, configuration, blocks, cacheTag);
```

Comment:
I don't quite get this change. It essentially turns off LLAP data caching of Parquet bytes, I'm not sure if this was intentional.